pkg/hub/managedcluster/manifests/managedcluster-clusterrole.yaml

```diff
@@ -1,22 +1,22 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: system:open-cluster-management:spokecluster:{{ .SpokeClusterName }}
+  name: system:open-cluster-management:managedcluster:{{ .ManagedClusterName }}
 rules:
 # Allow spoke agent to rotate its certificate
```

Comment:
OK for a follow-up PR: it might be easier to just remove `spoke` from `spoke agent` and just say `agent` in these comments.